src/com/stuypulse/stuylib/network/limelight/LimelightTable.java

```diff
@@ -74,57 +77,57 @@ public LimelightTable(String tableName) {
     /****************************************************************/
 
     // Whether the limelight has any valid targets (0 or 1)
-    public final NetworkTableEntry validTarget;
+    public final DoubleEntry validTarget;
 
     // Horizontal Offset From Crosshair To Target (-27 degrees to 27 degrees)
-    public final NetworkTableEntry xAngle;
+    public final DoubleEntry xAngle;
 
     // Vertical Offset From Crosshair To Target (-20.5 degrees to 20.5 degrees)
-    public final NetworkTableEntry yAngle;
+    public final DoubleEntry yAngle;
 
     // Target Area (0% of image to 100% of image)
-    public final NetworkTableEntry targetArea;
+    public final DoubleEntry targetArea;
 
     // Skew or rotation (-90 degrees to 0 degrees)
-    public final NetworkTableEntry targetSkew;
+    public final DoubleEntry targetSkew;
 
     // The pipeline’s latency contribution (ms) Add at
     // least 11ms for image capture latency.
-    public final NetworkTableEntry latency;
+    public final DoubleEntry latency;
 
     // Pixel information returned from these functions
-    public final NetworkTableEntry shortestSideLength;
-    public final NetworkTableEntry longestSideLength;
-    public final NetworkTableEntry horizontalSideLength;
-    public final NetworkTableEntry verticalSideLength;
+    public final DoubleEntry shortestSideLength;
+    public final DoubleEntry longestSideLength;
+    public final DoubleEntry horizontalSideLength;
+    public final DoubleEntry verticalSideLength;
 
-    // Corner Entries
-    public final NetworkTableEntry xCorners;
-    public final NetworkTableEntry yCorners;
+    // Corner DoubleArrayEntry
+    public final DoubleArrayEntry xCorners;
+    public final DoubleArrayEntry yCorners;
 
-    // Solve 3D Entry
-    public final NetworkTableEntry solve3D;
+    // Solve 3D DoubleArrayEntrys
+    public final DoubleArrayEntry solve3D;
 
-    // Camera Control Entries
-    public final NetworkTableEntry ledMode;
-    public final NetworkTableEntry camMode;
-    public final NetworkTableEntry pipeline;
-    public final NetworkTableEntry getPipeline;
-    public final NetworkTableEntry cameraStream;
-    public final NetworkTableEntry snapshotMode;
+    // Camera Control DoubleEntrys
+    public final DoubleEntry ledMode;
+    public final DoubleEntry camMode;
+    public final DoubleEntry pipeline;
+    public final DoubleEntry getPipeline;
+    public final DoubleEntry cameraStream;
+    public final DoubleEntry snapshotMode;
```

Comment:
These should. be integer entry ;(

Reply:
maybe
...

Reply:
i dont know whether these should be integer entries or not because like
they only store integers but Limelight is stupid and will publish doubles for booleans so
you never know

Reply:
That makes sense. Presumably they publish integers unless their documentation is lying.